praas-app-ionic/src/components/FormFieldWithError/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { IonItem } from '@ionic/react';
+import { FieldError } from 'react-hook-form';
+
+type Props = {
+  children: React.ReactNode;
+  error: FieldError | undefined;
+};
+
+const FormFieldWithError = ({ children, error }: Props) => (
+  <IonItem>
+    {children}
+    {error && <div className="error">{error.message}</div>}
```

Comment:
Move the error outside `IonItem`. It otherwise look weird. The line underneath  normally depict the Input field.

<img width="414" alt="image" src="https://user-images.githubusercontent.com/62667177/80317048-c209fa00-8801-11ea-9349-ed219d2ec88a.png">

Reply:
Hmm.. interestingly it wasnt visible in dark mode. My bad :D